src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -1131,8 +1140,14 @@ function showModalWarningIfSessionClosingSoon() {
 }
 
 function showModalSuccessIfResponsesSubmitted() {
-    if (hasSuccessMessage()) {
-        showModalAlert(getSuccessMessage(), RESPONSES_SUCCESSFULLY_SUBMITTED, null, BootstrapContextualColors.SUCCESS);
+    if (hasSuccessMessage() && hasIncompleteQuestionMessage()) {
+        const enlargedImportantIcon = '<span style="font-size:250%"> &#10071; </span>';
+        const incompleteMessage = enlargedImportantIcon + getIncompleteQuestionMessage();
+        showModalAlert(getSuccessMessage(), incompleteMessage + RESPONSES_SUCCESSFULLY_SUBMITTED,
+                null, BootstrapContextualColors.SUCCESS);
+    } else if (hasSuccessMessage()) {
+        showModalAlert(getSuccessMessage(), RESPONSES_SUCCESSFULLY_SUBMITTED,
+                null, BootstrapContextualColors.SUCCESS);
```

Comment:
Its possible to simplify this entire function to 

``` javascript
function showModalSuccessIfResponsesSubmitted() {
    const enlargedImportantIcon = '<span style="font-size:200%"> &#10071; </span>';
    const incompleteMessage = enlargedImportantIcon + getIncompleteQuestionMessage();

    if(hasSuccessMessage()) {
        showModalAlert(getSuccessMessage(),
            (hasIncompleteQuestionMessage() ? incompleteMessage : "") + RESPONSES_SUCCESSFULLY_SUBMITTED,
            null, BootstrapContextualColors.SUCCESS);
    }
}
```

Reply:
ohh thank you. that didnt cross my mind but it works really well!